<launch>
  <node pkg="calib_wide_camera" type="vis" name="vis" output="screen">
    <param name="file_path" type="string"
      value="/media/sam/CR7/huawei/extrinsic/close/0/"/>
    <param name="view_cloud" type="bool" value="1"/>
    <param name="view_ours" type="bool" value="1"/>
    <param name="view_gt" type="bool" value="0"/>
    <param name="view_orig" type="bool" value="0"/>
    <param name="write_pc" type="bool" value="0"/>
    <param name="write_bag" type="bool" value="0"/>
    <param name="kitti2pose" type="bool" value="0"/>
    <param name="pose2evo" type="bool" value="0"/>
    <param name="downsample_size" type="double" value="0.01"/>
    <param name="pcd_name_fill_num" type="int" value="0"/>
    <param name="marker_size" type="double" value="0.1"/>
  </node>

  <arg name="rviz" default="1"/>
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
    args="-d $(find pyramid)/rviz_cfg/rosbag.rviz" />
  </group>
</launch>